test/cluster/random_failures/test_random_failures.py

```diff
@@ -75,8 +83,12 @@ async def four_nodes_cluster(manager: ManagerClient) -> None:
                 "snapshot_trailing": "1",
             }
         )
+
     await wait_for_token_ring_and_group0_consistency(manager=manager, deadline=time.time() + 30)
 
+def get_random_rack():
+    rack_id = random.randint(1, 3)
+    return f"rack{rack_id}"
 
```

Comment:
Please, follow the PEP-8 code formatting conventions: https://peps.python.org/pep-0008/#blank-lines

tl;dr: "Surround top-level function and class definitions with two blank lines."

Also, looks like you can move this function to `cluster_events.py` and use it there too.